src/store/index.js

```diff
@@ -51,11 +20,18 @@ function _configureStore(aRootReducer, history, singleClusterMode) {
     });
 }
 
-export default function configureStore(aRootReducer = rootReducer, singleClusterMode = true) {
+function configureStore(aRootReducer = rootReducer, singleClusterMode = true) {
     ({backend, basename, clusterName} = getUrlData(window.location.href, singleClusterMode));
     const history = createBrowserHistory({basename});
 
     const store = _configureStore(aRootReducer, history, singleClusterMode);
     listenForHistoryChange(store, history);
     return {history, store};
 }
+
+export const webVersion = window.web_version;
+export const customBackend = window.custom_backend;
+
+export * from "./reducers"
```

Comment:
now do we really need to export everything from reducers here?

Reply:
I did this for future convenience. When we rewrite store to TS, type `IRootState` will be exported from there, not from reducers. So we won't need to update all the imports.

I updated exports, so `"./reducers"` now exports only `IRootState` type as named exports.